src/helpers/envutil.js

```diff
@@ -21,3 +30,24 @@ export function isNode() {
 export function workersTimeout(defaultValue = 0) {
   return (env && env.workerTimeout) || defaultValue;
 }
+
+export function blocklistUrl() {
```

Comment:
these functions are better named with `get` prepended.

Reply:
`get` doesn't work on Workers is what Santhosh told me...

Reply:
not JS getter, but naming the function like `getBlocklistUrl()`

Reply:
In a util? `get` is aloof here since there is no `set`. Prefixes usually reserved for information hiding (private / protected vars)

Reply:
there doesn't have to be a `setBlocklistUrl()`, for `getBlocklistUrl()` to exist.

Something named `blocklistUrl` is imperatively a variable containing blocklist url, not a function. Similar naming problem existed in another file when a constant `dnsCacheSize` is also a function name (which you have fixed in last commit I believe).

Reply:
Getter don't *need* Setters to exist...but this one isn't even a Getter. Even then, `get`/`set` prefix is a strictly Java convention. Kotlin doesn't follow it. Nor does c#. JS codebase do have those, but aren't needed here since there are no `private` / `protected` vars in use here. Nor, is there a `Class`.

See also: The `okhttp` API style (free of `get` / `set` prefixes despite being a Java lib): https://square.github.io/okhttp/4.x/okhttp/okhttp3/-response/